client/connect-account-page/payment-methods.tsx

```diff
@@ -38,7 +39,11 @@ const PaymentMethods: React.FC = () => {
 				<UnionPayIcon />
 				<JCBIcon />
 				<AffirmIcon />
-				<AfterpayIcon />
+				{ 'GB' === wcpaySettings?.connect?.country ? (
+					<ClearpayIcon />
+				) : (
+					<AfterpayIcon />
+				) }
```

Comment:
We need to use the store country here because the store is not connected to Stripe at this point